### PR TITLE
Feature/05/player input movement functionality

### PR DESCRIPTION
**Pull request by Muhammed-als**

### CODE REVIEW

File: Common/src/main/java/dk/sdu/sesem4/common/SPI/MovementControllerSPI.java
Comment (jakob753951) [inline Common/src/main/java/dk/sdu/sesem4/common/SPI/MovementControllerSPI.java:1]: Unnecessary formatting changes

File: Common/src/main/java/dk/sdu/sesem4/common/data/gamedata/GameData.java
Comment (jakob753951) [inline Common/src/main/java/dk/sdu/sesem4/common/data/gamedata/GameData.java:1]: More unnecessary formatting changes

File: Common/src/test/java/dk/sdu/sesem4/common/data/controllerParts/ControlTypeTest.java
Comment (jakob753951) [inline Common/src/test/java/dk/sdu/sesem4/common/data/controllerParts/ControlTypeTest.java:1]: This class seems unnecessary

**Review by jakob753951 — CHANGES_REQUESTED** [review]
Maybe remove ControlTypeTest. Other than that, it looks great :)

**Review by jakob753951 — APPROVED** [review]
LGTM

File: Common/src/test/java/dk/sdu/sesem4/common/data/controllerParts/ControlTypeTest.java
Comment (Muhammed-als) [reply Common/src/test/java/dk/sdu/sesem4/common/data/controllerParts/ControlTypeTest.java:1]: This class is removed

**Review by MGertz — APPROVED** [review]
